kolibri/plugins/learn/assets/src/views/LibraryPage.vue

```diff
@@ -0,0 +1,257 @@
+      EmbeddedSidePanel,
+      CategorySearchModal,
+    },
+    mixins: [commonLearnStrings, commonCoreStrings, languageSwitcherMixin, responsiveWindowMixin],
+    data: function() {
+      return {
+        showSearchModal: null,
+        currentCategory: '',
+        currentViewStyle: 'card',
+      };
+    },
+    computed: {
+      ...mapState('recommended', ['nextSteps', 'popular', 'resume']),
+      ...mapState('topicsRoot', { channels: 'rootNodes' }),
+      // screenLevel() {
+      //   if (window.innerWidth < 480) {
+      //     return 0;
+      //   } else if (window.innerWidth > 480 && window.innerWidth < 600) {
+      //     return 2;
+      //   } else if (window.innerWidth > 600 && window.innerWidth < 840) {
+      //     return 2;
+      //   } else if (window.innerWidth > 840 && window.innerWidth < 960) {
+      //     return 3;
+      //   } else {
+      //     return 4;
+      //   }
+      // },
+      carouselLimit() {
+        return this.windowIsSmall ? mobileCarouselLimit : desktopCarouselLimit;
+      },
+      trimmedPopular() {
+        return this.popular.slice(0, this.carouselLimit);
+      },
+      trimmedNextSteps() {
+        return this.nextSteps.slice(0, this.carouselLimit);
+      },
+      trimmedResume() {
+        return this.resume.slice(0, this.carouselLimit);
+      },
+      displayingSearchResults() {
+        return false;
+      },
+    },
+    created() {
+      if (this.$store.getters.isUserLoggedIn) {
+        const contentNodeIds = uniq(
+          [...this.trimmedNextSteps, ...this.trimmedPopular, ...this.trimmedResume].map(
+            ({ id }) => id
+          )
+        );
+
+        if (contentNodeIds.length > 0) {
+          ContentNodeProgressResource.fetchCollection({ getParams: { ids: contentNodeIds } }).then(
+            progresses => {
+              this.$store.commit('recommended/SET_RECOMMENDED_NODES_PROGRESS', progresses);
+            }
+          );
+        }
+      }
+    },
+    methods: {
+      genContentLink(id, isLeaf) {
+        return {
+          name: isLeaf ? PageNames.TOPICS_CONTENT : PageNames.TOPICS_TOPIC,
+          params: { id },
+          query: {
+            last: this.$store.state.pageName,
+          },
+        };
+      },
+      genChannelLink(channel_id) {
+        return {
+          name: PageNames.TOPICS_CHANNEL,
+          params: { channel_id },
+        };
+      },
+      handleShowSearchModal(value) {
+        this.currentCategory = value;
+        this.showSearchModal = true;
+      },
+      hideSearchModal() {
+        this.showSearchModal = false;
+      },
+
+      toggleCardView(value) {
+        this.currentViewStyle = value;
+      },
+    },
+    $trs: {
+      recent: {
+        message: 'Recent',
+        context:
+          'Header for the section in the Library tab with resources that the learner recently engaged with.',
+      },
+      /* eslint-disable kolibri/vue-no-unused-translations */
+      accessibility: {
```

Comment:
This could also be moved into `commonCoreStrings`, as it is used in at least 2 components. All accessibility sub-category items (resource properties) are already there, makes sense for the parent category to be too.

Reply:
Can update this.